updated sso screenshot (#8656)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-versions/release-notes.md

## February 2026

> - **Enhancement:** After a user accepts an email [invite](/docs/cloud/manage-access/invite-users) to access an [SSO-protected](/docs/cloud/manage-access/sso-overview) <Constant name="dbt_platform"/> account, the UI now explains that they must log in using SSO to fully redeem the invite and access the account. This replaces the previous "Joined successfully" message and helps avoid confusion when users accept an invite but do not complete the SSO login flow.

Reviewer: ```suggestion
- **Enhancement:** After a user accepts an email [invite](/docs/cloud/manage-access/invite-users) to access an [SSO-protected](/docs/cloud/manage-access/sso-overview) <Constant name="dbt_platform"/> account, the UI now prompts them to log in with SSO to complete the process. This replaces the previous "Joined successfully" message, helping avoid confusion when users accept an invite but do not complete the SSO login flow.
```

Reviewer: Hiya @mirnawong1 

This is also a non-blocker, but i've just reworded this slightly.

Kind Regards
Natalie

Author: good stuff! fixed up the apostrophes since they were italicized for some reason
